### DISCUSSION TITLE
Reveal.js Presentation: how to set background for slides without title?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#605] [Q&A] [answered]
Hi,
in the manual I have "You can apply full page backgrounds outside of the slide area by adding a background attribute to your slide headers", then I must a title, something like `## Slide Title {background-color="aquamarine"}`

How to apply a background color to a slide without any title?

Thank you

**Comment by cderv** [comment, marked as answer]
You can do this by using empty header
````markdown
---
title: "A title"
author: "me"
format:
  revealjs:
    theme: dark
---

## {background-color="aquamarine"}

Hello World

## Lorem ipsum

````

**Reply by aborruso** [reply]
Thank you very much.

Does it make sense to you that I open a PR, to add a note in the documentation?

**Reply by cderv** [reply]
You could open a PR but I am not sure where is the best place to add this. This will happen not only with `background-color` but any attributes that should be on header. Maybe have something about no-title slide 🤔 

But thanks for opening this question as it will at least serve as documentation if anyone else have the same question.

**Reply by cscheid** [reply]
I think we could add a clarification right where @aborruso started, on https://quarto.org/docs/presentations/revealjs/#slide-backgrounds, since this is likely to be a common case for it to come up.

**Reply by cscheid** [reply]
The [clarification is up](https://quarto.org/docs/presentations/revealjs/#slide-backgrounds-without-a-title). Thanks for the suggestion!

**Reply by aborruso** [reply]
> The [clarification is up](https://quarto.org/docs/presentations/revealjs/#slide-backgrounds-without-a-title). Thanks for the suggestion!

You and this community are very kind.

Thank you

**Reply by cderv** [reply]
Awesome ! Thanks @cscheid !
